README.md

```diff
@@ -47,4 +66,5 @@ Logs for the webserver are located in
 `/usr/local/openresty/nginx/logs/error.log`
 `/usr/local/openresty/nginx/logs/access.log`
 
-These logs are not rotated and will eventually fill the 8GB disk.
+These logs are rotated with a manuall provisioned `/etc/logrotate.d/openresty`
```

Comment:
nit: manually

Reply:
Aha, thanks for catching that, I've fixed it in 79d6f55903d36fc9f0ec8ad092d276a140b58f4b